Content.Server/PAI/PAISystem.cs

```diff
@@ -129,7 +129,7 @@ private void UpdatePAIAppearance(EntityUid uid, PAIStatus status)
 
         private void AddWipeVerb(EntityUid uid, PAIComponent pai, GetActivationVerbsEvent args)
         {
-            if (args.User == null || !args.CanAccess || !args.CanInteract)
```

Comment:
why remove this check?

Reply:
Just because user is not nullable, nor should it be. A verb needs a user to perform the verb.

In general I was looking at warnings about non-nullable variables being equated to null, because that often indicated locations where a nullable IEntity was just replaced with a default non-nullable entityUid.

It seems I've actually missed a few as well, e.g. `InternalsComponent.BreathToolEntity` still has this issue.